src/faro/faroEnv.ts

```diff
@@ -0,0 +1,38 @@
+import { type Environment } from './getEnv';
+import { type FaroEnvironment } from './getFaroEnv';
+
+export const FARO_ENVIRONMENTS = new Map<Environment, FaroEnvironment>([
+  // Uncomment this map entry to test from your local machine
+  // [
+  //   'local',
+  //   {
+  //     environment: 'local',
+  //     appName: 'grafana-logsdrilldown-app-local',
+  //     faroUrl: 'https://faro-collector-ops-eu-south-0.grafana-ops.net/collect/54fb57556eb45f8857edb9f0345c0d7a',
```

Comment:
i think we listen on `http://localhost:12347/collect` right now for local faro

Reply:
also I'd like to instrument this so it only sends when the docker-compose.local.yaml is kicking things off instead of requiring code changes